src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -461,10 +518,15 @@ void GcsPlacementGroupManager::HandleGetPlacementGroup(
     GCS_RPC_SEND_REPLY(send_reply_callback, reply, Status::OK());
   };
 
-  Status status =
-      gcs_table_storage_->PlacementGroupTable().Get(placement_group_id, on_done);
-  if (!status.ok()) {
-    on_done(status, boost::none);
+  auto it = registered_placement_groups_.find(placement_group_id);
```

Comment:
this looks like a recurring pattern? should we create a helper function for it?

Reply:
which part are you talking about? 
```cpp
 auto it = registered_placement_groups_.find(placement_group_id);
if (it != registered_placement_groups_.end()) {
```
=> this one?

Reply:
I just used `registered_placement_groups_ .contains`. Lmk if you meant something else